src/App.js

```diff
@@ -0,0 +1,12 @@
+import './App.css';
+import TodoList from './components/TodoList';
+
+function App() {
+  return (
```

Comment:
-[] Optional :- Consider using an arrow function instead. It will improve how the code looks and make it easier for users to understand. 💯